README.md

```diff
@@ -5,10 +5,14 @@ Azure Cloud Shell is a browser-based shell environment which enables Azure custo
 
 For more details, check out [Overview of Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/overview#:~:text=Features%201%20Browser-based%20shell%20experience.%20...%202%20Choice,7%20Connect%20your%20Microsoft%20Azure%20Files%20storage.%20).
 
-Try out Cloudshell by clicking the button below.
+Try out Cloud Shell by clicking the button below.
 
 [![](https://shell.azure.com/images/launchcloudshell.png "Launch Azure Cloud Shell")](https://shell.azure.com)
 
+Try out Cloud Shell locally by simply pulling from the Microsoft Container Registry: mcr.microsoft.com/azure-cloudshell:latest
+
+*Please note that the image from the Azure portal or the Microsoft Container Registry may not directly reflect what is in this GitHub repository.

```

Comment:
I think we should expand a bit on what the differences are. "they might be different" doesn't help much.

Reply:
Is the ``mcr.microsoft.com/azure-cloudshell`` container image syndicated to Docker Hub like many of the other Microsoft Container Registry images? I tried searching for it today and couldn't find it. Wasn't sure if it was not included by design or waiting for propagation.

> **Example**: ``mcr.microsoft.com/dotnet/sdk:5.0`` can be found syndicated on Docker Hub at <https://hub.docker.com/_/microsoft-dotnet-sdk/>.

Reply:
@seesharprun Currently, they are not syndicated to Docker Hub.